### ISSUE TITLE
Dragging a tab makes the window slowly scroll up for a while even after not dragging the tab anymore

### ISSUE THREAD

**Issue by RuurdBijlsma** [microsoft/terminal#9109]
# Environment

```none
Windows build number: 10.0.19042.0
Windows Terminal version (if applicable): 1.5.10271.0
```

# Steps to reproduce

1. Have a single tab open
2. Drag that tab downward a little (starting your drag at the bottom of the tab, sometimes text in the terminal gets selected this way, but that's not necessary for this to happen)

# Expected behavior

It shouldn't scroll up forever when you do that

# Actual behavior

If the terminal can scroll up it will scroll up slowly and go on until you click in the terminal
video: https://www.youtube.com/watch?v=o0T1jeFemvQ

**Comment by zadjii-msft** [comment]
Well that certainly looks insane. I can't get this to repro at all though, unfortunately. Who knows, maybe this will be made better with #1256, or _far far worse_. I guess we'll leave this open on the backlog for now.

**Comment by leduyquang753** [comment]
This looks like it has something to do with the auto-scroll behavior when dragging and dropping: when an object is being dragged by the cursor and the cursor is near an edge of a scrollable container, the container will think the user is wanting to scroll to reveal the portion of the container that the user wants to drop the object to. And Windows terminal's console container is also doing that except that it doesn't stop when the mouse has been released.

![Screenshot at 12h47 20 23-04-2021](https://user-images.githubusercontent.com/26449060/115824406-10ac9d00-a432-11eb-94c6-3268b95ae709.gif)

**Comment by zadjii-msft** [comment]
> the auto-scroll behavior when dragging and dropping

Is there some sort of Windows setting that controls that? Because we certainly didn't implement anything like that for the Terminal 😄 We do support "dragging a selection outside the control will scroll the control", but nothing for "dragged something over the edge of the control will scroll the control".



Holy crap I got it to repro. I figured out what's different between mine & OP's setup - you need to drag the tab into the _padding_. If you're in the padding, then we'll try and start an autoscroll, even without a selection. 

This'll be a trivial fix once #9820 lands
